WalletWasabi/Models/SerializableException.cs

```diff
@@ -25,16 +25,16 @@ public SerializableException(Exception ex)
 		}
 
 		[JsonProperty(PropertyName = "ExceptionType")]
-		public string ExceptionType { get; set; }
+		public string? ExceptionType { get; private set; }
```

Comment:
`private set` -> `init`?

Reply:
Then you can have constructor with `[JsonConstructor]` that constructs the object correctly and have `init`.

Reply:
I added [JsonConstructor]. If the order of properties is changed in the class then the constructor will not work correctly right?

Reply:
It should work regardless of the order of properties in code.

Reply:
How? I guess it is calling the [JsonConstructor] and has no mapping about the constructor parameters.

Reply:
https://docs.microsoft.com/en-us/dotnet/standard/serialization/system-text-json-how-to?pivots=dotnet-5-0#deserialization-behavior indicates that matching of JSON keys and .NET properties is done by name and not order. I expect the same thing for `[JsonConstructor]` but I don't have a proof.

Reply:
ACK. We use [JsonConstructor] elsewhere in Wasabi so it should be fine. It is cleaner than the empty constructor.